### ISSUE TITLE
Enabling jupyterlab-quarto plugin prevents Markdown output from displaying

### ISSUE THREAD

**Issue by evanlesmez** [quarto-dev/quarto#201]
Python version 3.10.6  
Steps:  
```sh
python -m venv env
source env /bin/activate
python3 -m pip install jupyterlab jupyterlab-quarto
jupyter lab
```

![jlab-quarto-md-output-error](https://github.com/quarto-dev/quarto/assets/31675729/944ccd52-0d73-420f-9428-f30aeebe094c)

**Comment by jjallaire** [comment]
@dragonstyle Could you take a look at this?

**Comment by michaelfsp** [comment]
I'm experiencing the same problem. Have also noticed that this happens with JupyterLab 4.0.2 but not with JupyterLab 3.6.4.

**Comment by dragonstyle** [comment]
Yes I can confirm that this is an incompatibility between the current version of the extension and JupyterLab 4.0. I will be working on getting 4.0 support added shortly!
